src/main/java/seedu/address/model/packinglistitem/UniquePackingList.java

```diff
@@ -117,7 +117,7 @@ public void setItem(Item target, Item editedItem) {
      *
      * @param replacement the replacement
      */
-    public void setPackingList(UniquePackingList replacement) {
+    public void setPackingLists(UniquePackingList replacement) {
```

Comment:
`setPackingList` already means multiple items `setPackingLists` means multiple lists which isn't the intention here


**Suggestions:**
Either (preferred since the above method is `setItem()`)
```suggestion
    public void setItems(UniquePackingList replacement) {
```
Or
```suggestion
    public void setPackingList(UniquePackingList replacement) {
```

Reply:
Ill just change it to setPackingList for now. Need to look at cat to see which one she prefers.